tool/linter_rules/exclusion_reasons.json

```
@@ -14,18 +14,19 @@
   "do_not_use_environment": "Not specified",
   "document_ignores": "Not specified",
   "invalid_runtime_check_with_js_interop_types": "[Pending recommendation](https://github.com/dart-lang/lints/issues/188)",
+  "library_names": "[Unnecessary](https://github.com/dart-lang/lints/issues/172)",
```

I think that having "unnecessary" as a reason sounds too imposing, what about also adding a reference to `unnecessary_library_name` to soothe it out?

```suggestion
  "library_names": "[Unnecessary](https://github.com/dart-lang/lints/issues/172) and superseded by `unnecessary_library_name`](https://github.com/dart-lang/lints/issues/172)",
```

---

Same would go for in the README right?

---

Indeed, the README is automatically generated from this file so it would just take running a command and formatting.